test/malli/core_test.cljc

```diff
@@ -204,6 +204,18 @@
                      [:x boolean?]
                      [[:opt :y] int?]
                      [[:req :z] string?]])
+          schema3 (m/schema
+                    [:map keyword? int?])
+          schema4 (m/schema
+                    [:map
+                     [keyword? int?]
+                     [:x string?]
+                     [:y int?]])
+          schema5 (m/schema
+                    [:map
+                     [:x string?]
+                     [[:opt :y] int?]
+                     [keyword? int?]])
```

Comment:
there should be a test that for multiple predicate keys too. Schema explicitly only allows one extra key per map, I think we might not need that restriction.

Reply:
What do you mean?

Reply:
My bad, was thinking [Plumatic schema](https://github.com/plumatic/schema), which has the limitation - just one predicate key per map

Reply:
Allow something like that?
```clojure
(m/schema
  [:map
    [keyword? int?]
    [keyword? string?]])
```

Reply:
Let's not. That can already be defined as:

```clj
[:map
  [keyword? [:or int? string?]]
```
... and this would cause trouble when later doing tools to merge maps and removing `keyword?` key might or might not remove all keys. Same actually applies to normal keys, this should be illegal:

```clj
[:map [:x int?] [:x string?]]
```